README.md

```diff
@@ -1,7 +1,41 @@
-# TCET Open Source Documentation Team
+# TCET Open Source Documentation Website
 
-One of the key visions of the TCET Open Source Organization is to provide users of our software and others willing to contribute with access to project documentation for all of our current projects. The organization recognizes the importance of clear and concise documentation when creating open-source software, to ensure it's development is rapid, continous & traceble.
+This repository contains the source code of the Documentation Website.
 
-The "Documentation Website" will not only have our project documentation, but also blogs. It will also act as our Organization's Website.
+One of the key visions of the TCET Open Source Organization is to provide users of our software and others willing to contribute with access to project documentation for all of our current projects. The organization recognizes the importance of clear and concise documentation when creating open-source software, to ensure it's development is rapid, continous and traceble.
 
-This is the source code of the Documentation Website.
\ No newline at end of file
+The **Documentation Website** will act as our Organization's website. It will consist of the *documentation of all projects* undertaken by TCET Open Source. The website will also have dedicated *blogs contributed by the community* covering various different interests.
+
+## Cloning
+
+To clone the repository copy paste the following command in your system's terminal.
+
+```
+git clone https://github.com/tcet-opensource/documentation.git
+```
+
+## Project Setup
+
+This command is necessary for ensuring all dependencies are present on your local system.
+
+```
+npm install
+```
+
+## Project Testing
+
+Run the project live on your local system to make changes and check the updates.
+
+```
+npm run start
+```
+
+<div align="center">
+<h3> Connect with us<a href="https://gifyu.com/image/Zy2f"><img src="https://github.com/milaan9/milaan9/blob/main/Handshake.gif" width="50px"></a>
+</h3> 
+<p align="center">
+    <a href="https://www.linkedin.com/company/tcet-opensource/" target="_blank"><img alt="LinkedIn" width="40px" src="https://cdn-icons-png.flaticon.com/512/3536/3536505.png"></a> &nbsp&nbsp&nbsp
```

Comment:
can we put linkedin after all of these? also I think we should add discord link to the discussion channel

Reply:
Added Discord invite and re-arranged embed icons.